content/concepts/protocols.md

```diff
@@ -1,227 +1,167 @@
-| `/ipfs/id/push/1.0.0` | same as [identify above](#identify) |
-
-A slight variation on `identify`, the `identify/push` protocol sends the same `Identify` message, but it does so proactively
-instead of in response to a request.
-
-This is useful if a peer starts listening on a new address, establishes a new [relay circuit](/concepts/circuit-relay/), or
-learns of its public address from other peers using the standard `identify` protocol. Upon creating or learning of a new address,
-the peer can push the new address to all peers it's currently aware of. This keeps everyone's routing tables up to date and
-makes it more likely that other peers will discover the new address.
-
-### kad-dht
-
-`kad-dht` is a [Distributed Hash Table][wiki_dht] based on the [Kademlia][wiki_kad] routing algorithm, with some modifications. 
-
-libp2p uses the DHT as the foundation of its [peer routing](/concepts/peer-routing/) and [content routing](/concepts/content-routing/) functionality. To learn more about DHT and the Kademlia algorithm,
-check out the [Distributed Hash Tables guide][dht] on the IPFS documentation site. In addition, check out the [libp2p implementations page](https://libp2p.io/implementations/) for updates on all the kad-libp2p implementations.
-
-<!-- Consider adding general kad matrix on implementations page, then link -->
-
-[spec_kad]: https://github.com/libp2p/specs/pull/108
-
-[wiki_dht]: https://en.wikipedia.org/wiki/Distributed_hash_table
-[wiki_kad]: https://en.wikipedia.org/wiki/Kademlia
-[dht]: https://docs.ipfs.tech/concepts/dht/
-
-### Circuit Relay
-
-| **Protocol id**               | spec                             |                | implementations |
-|-------------------------------|----------------------------------|----------------|-----------------|
-| `/libp2p/circuit/relay/0.1.0` | [circuit relay spec][spec_relay] | [go][relay_go] | [js][relay_js]  |
-
-[spec_relay]: https://github.com/libp2p/specs/tree/master/relay
-[relay_js]: https://github.com/libp2p/js-libp2p-circuit
-[relay_go]: https://github.com/libp2p/go-libp2p-circuit
-
-As described in the [Circuit Relay article](/concepts/circuit-relay/), libp2p provides a protocol
-for tunneling traffic through relay peers when two peers are unable to connect to each other
-directly. See the article for more information on working with relays, including notes on relay
-addresses and how to enable automatic relay connection when behind an intractable NAT.
-
-<!-- links -->
-
-[definition_switch]: /reference/glossary/#switch
-[definition_multiaddr]: /reference/glossary/#multiaddr
+{{% notice "note" %}}
+Check out the [libp2p implementations page](https://libp2p.io/implementations/) for 
+updates on all the libp2p implementations.
+{{% /notice %}}
 
-[repo_multistream-select]: https://github.com/multiformats/multistream-select
+| **Specification**                                                                          | **Protocol ID**                    |
```

Comment:
We should remove all transport-related protocols. They're (soon) negotiated differently, and don't really fit the above description of protocols anyway. That means removing: mDNS, mplex, Noise, Plaintext, pnet, QUIC, TLS 1.3 and WebTransport.